sharding/handler/log_handler.py

```diff
@@ -1,54 +1,42 @@
 import logging
 
-from sharding.handler.utils.web3_utils import (
-    get_recent_block_hashes,
-    get_canonical_chain,
-)
-
 
 class LogHandler:
 
-    logger = logging.getLogger("evm.chain.sharding.LogHandler")
-
-    def __init__(self, w3, history_size=256):
-        self.history_size = history_size
-        self.w3 = w3
-        # ----------> higher score
-        self.recent_block_hashes = get_recent_block_hashes(w3, history_size)
-
-    def get_new_logs(self, address=None, topics=None):
-        # TODO: should see if we need to do something with revoked_hashes
-        #       it seems reasonable to revoke logs in the blocks with hashes in `revoked_hashes`
-        revoked_hashes, new_block_hashes = get_canonical_chain(
```

Comment:
I see that these codes of finding `recent_block_hashes` are removed and the purpose of `get_logs` is different from `get_new_logs`. I guess it's relevant to how we proceed windback efficiently in the future stage. cc @mhchia to check if it's okay to remove them.

Reply:
Since in a fork-free shard chain design we don't need to store the logs in order to determine the head of the shard chain, we therefore don't need to worry about re-org of main chain.
And so in my opinion it's okay to remove the flow of handling re-org here.

And now `LogHandler` seems pretty simple and straightforward(It just proxy the `get_logs` call to `web3`.), maybe we can just merge it into `ShardTracker`.

Reply:
I would say just remove `LogHandler` since it doesn't keep data anymore.